src/metadata_tool/tools/parsers.clj

```diff
@@ -21,35 +21,42 @@
             [hickory.select                       :as sel]
             [metadata-tool.sources.confluence     :as cfl]
             [metadata-tool.config                 :as cfg]
+            [clojure.pprint                       :as pp]
             [metadata-tool.sources.metadata       :as md]))
 
 (def not-nil? (complement nil?))
 
 (defn parse-string
   [s]
-  (-> s
-      (str/replace "&nbsp;" " ")
-      (str/replace "\u00A0" " ")
-      str/trim))
+  (if (empty? s)
+    ""
+    (-> s
+        (str/replace "&nbsp;" " ")
+        (str/replace "\u00A0" " ")
+        str/trim)))
+
+(defn resolve-acronym
+  [string]
+  (if-let [acronym (get (:acronyms (:meetings cfg/config)) string)]
```

Comment:
this is best left for another PR, but every namespace seems to know a lot about the internal structure of the global config singleton.  I'd consider moving all these accessors to the `cfg` namespace.